qlib/pagetable.rs

```diff
@@ -127,12 +129,23 @@ impl PageTables {
     }
 
     pub fn SwitchTo(&self) {
+        #[cfg(feature = "cc")]
```

Comment:
I personally prefer having two SwitchTo with different config terms, instead of one SwitchTo with two different behaviors.